packages/crypto/__tests__/handlers/transactions/transfer.test.js

```diff
@@ -18,14 +18,14 @@ describe('TransferHandler', () => {
       expect(handler.canApply).toBeFunction()
     })
 
-    it('should be ok', () => {
-      expect(handler.canApply(wallet, transaction)).toBeTrue()
+    it('should be truthy', () => {
+      expect(handler.canApply(wallet, transaction, [])).toBeTrue()
     })
 
-    it('should not be ok', () => {
+    it('should be falsy', () => {
```

Comment:
Should be described as `true` instead of `truthy` as `truthy` can mean multiple types that are empty rather then a `bool`.

Reply:
Changed